findbugs/etc/findbugs.xml

```diff
@@ -621,6 +621,13 @@
                     reports="DM_DEFAULT_ENCODING"/>
           <Detector class="edu.umd.cs.findbugs.detect.CheckRelaxingNullnessAnnotation" speed="fast"
                     reports="NP_METHOD_RETURN_RELAXING_ANNOTATION,NP_METHOD_PARAMETER_TIGHTENS_ANNOTATION"/>
+          <Detector class="edu.umd.cs.findbugs.detect.MonitoringValueBasedClass" speed="fast"
+                    reports="VBC_MO_LOCK,VBC_MO_WAIT,VBC_MO_NOTIFY"/>
+          <Detector class="edu.umd.cs.findbugs.detect.IdentityHashCodeOnValueBasedClass" speed="fast"
+                    reports="VBC_IDENTITY_HASHCODE"/>
+                    
+          <BugCode abbrev="VBC_MO_LOCK"/>
+          <BugCode abbrev="VBC_MO_WAIT"/>
```

Comment:
I see no need to declare separate abbreviations VBC_MO_LOCK, etc. Instead you should declare the bug patterns like:

    <BugPattern abbrev="VBC" type="VBC_MO_LOCK" category="BAD_PRACTICE"/>

And the same row should be added for other newly-defined patterns (`VBC_MO_WAIT`, `VBC_MO_NOTIFY`, `VBC_IDENTITY_HASHCODE`). The `abbrev` is somewhat a group of related bugs. I think, `VBC` is good for all new bug patterns you introduce.